src/app/journal/journal/class-journal/subject-journal/homework-bottom-sheet-overview.components.ts

```diff
@@ -133,7 +133,8 @@ export class HomeworkBottomSheetOverviewSheetComponent implements OnInit {
    * all the transformation logic is done it creates temporary link in order
    * to let user save the file itself
    */
-  dwnl() {
+  dwnl(event: Event) {
```

Comment:
use meaningful names

Reply:
when you add params, annotations must be changed as well

Reply:
Sorry, i will fix it with the next pull request